Vault Staking/chains.json

```diff
@@ -24,8 +24,17 @@
       "address": "evmosvaloper1k5hcl0cjprzewwvzhz3um589kx8skzlrneuepu",
       "restake": {
         "address": "evmos1emzdjlfj55gezwr7cswr2e7ys7nqarclle3kpk",
-        "run_time": "1 hour",
-        "minimum_reward": 1000000
+        "run_time": "2x per day",
```

Comment:
I've just realised you have invalid run times. It should be an array with two UTC times, e.g `["02:00","14:00"]`, or even just `every 12 hours`

Reply:
Good catch, I was wondering why it showed 'daily' on the website even though I had '2x per day' in here. Fixed!